### PR TITLE
Use referral link instead

### PR DESCRIPTION
**Pull request by cnevinc**

### CODE REVIEW

**Comment by chianger** [comment]
To enable VPN recommender we would need this URL also remote configurable.

File: app/src/main/java/org/mozilla/focus/home/FeatureSurveyViewHelper.kt [hunk 1]
```diff
@@ -82,8 +84,8 @@ class FeatureSurveyViewHelper internal constructor(private val context: Context,
                     // Dismiss view immediately
                     parentView.removeView(rootView)
                     isViewInit = false
-                    // VPN app is installed, go to google play store
-                    IntentUtils.goToPlayStore(context, Constants.PACKAGE_EXPRESS_VPN)
+                    // VPN app is installed, open Play Store in a new tab
```
Comment (mxlius) [inline app/src/main/java/org/mozilla/focus/home/FeatureSurveyViewHelper.kt]: I guess this comment should be "VPN app is NOT installed", so we open play store in a new tab?

File: firebase/src/firebase/java/org/mozilla/focus/utils/FirebaseWrapper.java [hunk 2]
```diff
@@ -74,12 +74,12 @@
 
 
     // get Remote Config string
-    static String getRcString(@NonNull Context context, @NonNull String key) {
```
Comment (mTwTm) [inline firebase/src/firebase/java/org/mozilla/focus/utils/FirebaseWrapper.java]: You might need to change the one in firebase_no_op/.../FirebaseWrapper.java as well.

**Comment by cnevinc** [comment]
Actually, this is where we put the default value
https://github.com/mozilla-tw/Rocket/blob/e193c2a13f8cda884176129f9f1e6b427c6ce3da/app/src/main/java/org/mozilla/focus/utils/FirebaseHelper.java#L251
Cause we don't want to open another interface to let client code setting the default value.

**Pull request merged**